modules/cms-kit/host/Volo.CmsKit.Web.Unified/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@abp/aspnetcore.mvc.ui.theme.basic": "^5.2.0-rc.1",
-    "@abp/cms-kit": "5.2.0-rc.1"
+    "@abp/cms-kit": "5.2.0-rc.1",
+    "@abp/codemirror": "~5.2.0-rc.1"
```

Comment:
no need add @abp/codemirror, it has reference by @abp/cms-kit (by @abp/cms-kit.admin internal)

Reply:
it's just for testing, because they haven't released yet

Reply:
I shouldn't push those I think